packages/react-app/theme.js

```diff
@@ -59,6 +59,11 @@ const theme = extendTheme({
     global: props => ({
       body: {
         color: props.colorMode === "light" ? "light.text" : "dark.text",
+        background:
+          props.colorMode === "light"
+            ? "linear-gradient(180deg, #BFCDFF 10.47%, #D9EBE5 60.39%, #EFFBCA 89.55%, rgba(239, 251, 202, 0.00) 116.3%)"
+            : "linear-gradient(180deg, #0B2041 10.47%, #2F3679 107%)",
```

Comment:
Did same at https://github.com/scaffold-eth/buidlguidl-v3/pull/95/commits/e04ec75259f39aca2b454eebbb2e425a22f7b6bb 🙌

I was initially trying to do it like this : https://twitter.com/chakra_ui/status/1640807319645859841 , but it seems it is only supported in v2 and in our case since we don't need to reuse it so its we inline it